library/src/main/java/com/chuckerteam/chucker/internal/support/NotificationHelper.kt

```diff
@@ -17,10 +17,14 @@ import com.chuckerteam.chucker.internal.ui.BaseChuckerActivity
 import java.util.HashSet
 
 class NotificationHelper(val context: Context) {
+
     companion object {
-        private const val CHANNEL_ID = "chucker"
```

Comment:
Not a review, just a side note. Changing the Channel ID here will cause the fix we suggested here #154 to do not work anymore. Something that we want to point out somewhere.

As an alternative I suggest we keep the `TRANSACTIONS_CHANNEL_ID` as `"chucker"` as it's probably the most noisy.

Reply:
I am not sure that this is kind of an issue which might required rolling back. 
It is better to have some meaningful names for all channels once we create them instead of having generic `chucker` for one and `chucker errors` for another.
Of course we will mention this change at least in #154